app/models/manifest_uploader.rb

```diff
@@ -79,4 +84,10 @@ def check_for_missing_or_invalid_data
       end
     end
   end
+
+  def check_if_any_labwares_are_locations
+    if labwares.compact.any? { |labware| labware.match(/^#{Location::BARCODE_PREFIX}?/o) }
```

Comment:
"Upload labwhere csv now has a check to make sure there are no labwares with the same barcode as an existing location"
-> This code doesn't check that any Locations exist with the uploaded Labware barcode, but just that the uploaded Labware barcode doesn't start with `lw-`
Which is fine, just wanted to be clear. 
This is similar to the scan_form, which apparently wans't working? Does that need updating too? The difference is `.compact`, which I'm not sure how that makes the difference?!

Reply:
We have already added the fix to the scan form and it is in production. I was unsure on this story but occassionally they upload a dodgy csv which has location barcodes in the plate barcode column. The compact removes any nils as it is possible to scan nothing which would just add a line feed or an extra blank row. We don't want to throw an error as it is not user errror.